python/sglang/srt/layers/moe/ep_moe/layer.py

```diff
@@ -74,6 +79,13 @@
     from aiter.fused_moe import fused_moe
     from aiter.ops.shuffle import shuffle_weight
 
+if _use_flashinfer_blockscale_fp8_moe:
+    try:
+        import flashinfer.fused_moe as fi_fused_moe
```

Comment:
QQ can I use `0.2.7.post1` to test this branch?

Reply:
Probably not. [This](https://github.com/flashinfer-ai/flashinfer/commit/bd74e15cce564c36adc5a49800f338b269759961#diff-d2eee7647631902c9b37dc7b03c56129e3f5b94f88df67cdf18edeff5c03d159) is the related commit for flashinfer and it's labelled `0.2.8`.